docs/preview/03-Features/sinks/azure-application-insights.md

```diff
@@ -174,35 +174,44 @@ ILogger logger = loggerConfig.CreateLogger();
 
 ### Q: Where can I initialize the logger in an ASP.NET Core application or other hosted service?
 
-Simply use the `UseSerilog` extension method on `IHostBuilder` which accepts an `Action<HostBuilderContext, IServiceProvider, LoggerConfiguration>`.
-This Action gives you access to the configuration and the configured services.
-
 If the connection string is stored as a secret in -for instance- Azure KeyVault, the `ISecretProvider` from [Arcus secret store](https://security.arcus-azure.net/features/secret-store) can be used to retrieve the connection string.
 
+Use the `UseSerilog` extension method on `IHostBuilder` which accepts an `ILogger` and use the Serilog's static `Log.Logger` property to setup the logger. This makes sure that we can reload the logger afterwards (`CreateBootstrapLogger`). This is needed because otherwise an endless loop will occur when the Arcus secret store is used (secret store required logging, logging requires secret store).
```

Comment:
```suggestion
Use the `UseSerilog` extension method on `IHostBuilder` which accepts an `ILogger` and use the Serilog's static `Log.Logger` property to setup the logger. 
This allows the Startup code to have preliminary logging before we can setup logging to Application Insights.  This bootstrap logger is used when setting up the Secret Store. When setting up logging to Application Insights, we use the Secret Store to retrieve the Application Insights connectionstring, and the Secret Store uses the bootstrap logger at that point.
The bootstrap logger allows to reload the logger after we've setup logging to Application Insights. 
```

Reply:
I'll use 'lower case' secret store, but yes, will alter this a bit. Thx!

Reply:
Changed in https://github.com/arcus-azure/arcus.observability/pull/446/commits/b969671242a2ecd195b0331dc47529f92174fc77

Reply:
> So it is not mandatory. However, while reading it, I thought it was mandatory.
As I read it, it looks like we must do that. So, maybe we should rephrase this a bit ?

Mandatory, well, it's recommended in the same way we promote the use of Arcus secret store, managed identity authentication, Serilog logging...